components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -219,10 +219,11 @@ private String processLogoutRequest(HttpServletRequest request, HttpServletRespo
                 .getParameter(OIDCSessionConstants.OIDC_POST_LOGOUT_REDIRECT_URI_PARAM);
         String state = request
                 .getParameter(OIDCSessionConstants.OIDC_STATE_PARAM);
+        String tenantDomain = request.getParameter(OIDCSessionConstants.TENANT_DOMAIN);
```

Comment:
do we need to rely on the tenantDomain query param?
Can't we derive the spTenantDomain from the clien_id?

Reply:
we already do this if 'isJWTSignedWithSPKey' property is set to true [1]. Otherwise we need to get user's tenant domain

[1] https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java#L325